src/Analysis/Codelyzer.Analysis.Build/Models/ProjectBuildResult.cs

```diff
@@ -22,6 +22,7 @@ public class ProjectBuildResult : IDisposable
         public List<string> TargetFrameworks { get; set; }
         public string ProjectGuid { get; set; }        
         public string ProjectType { get; set; }
+        public bool IsSyntaxAnalysis { get; set; }
```

Comment:
This property name is misleading because when it is `false`, syntax analysis is still being done. Consider renaming it to `IsSyntaxAnalysisOnly` or something that indicates that when this property is `true` then something else is not being done (semantic analysis in this case).